nautobot_netbox_importer/management/commands/import_netbox_objectchange_json.py

```diff
@@ -89,7 +89,14 @@ def process_objectchange(self, entry, options, used_error_messages):
                 self.logger.error(error_message)
             return
 
-        if not options["dry_run"]:
+        if (
+            not options["dry_run"]
+            and ObjectChange.objects.filter(
+                request_id=entry["fields"]["request_id"], time=entry["fields"]["time"]
+            ).count()
+            == 0
```

Comment:
I believe this is a more efficient approach:

```suggestion
            and ObjectChange.objects.filter(
                request_id=entry["fields"]["request_id"], time=entry["fields"]["time"]
            ).exists()
```